MedicalDocManagment.WebUI/Controllers/api/AdminController.cs

```diff
@@ -88,31 +88,56 @@ public IHttpActionResult GetPaged(int pageNumber = 1, int pageSize = 20)
 
 
         [HttpPut]
-        public async Task<IHttpActionResult> EditUser(UserEditModel userEditModel)
+        public async Task<HttpResponseMessage> EditUser(string id, UserEditModel userEditModel)
         {
             var user = await _usersManager.FindByIdAsync(userEditModel.Id);
-            Position newPosition = null;
 
             if (user == null)
             {
-                return NotFound();
-            }
-
-            //newPosition = _GetPositionById(userEditModel.Position.PositionId);
-
-            if (newPosition != null)
-            {
-                user.Position = newPosition;
+                return Request.CreateResponse(HttpStatusCode.NotFound, $"User wasn't found with ID {id}.");
             }
 
+            user.PositionId = userEditModel.PositionId;
+            user.UserName = userEditModel.UserName;
+            user.Email = userEditModel.Email;
             user.FirstName = userEditModel.FirstName;
             user.LastName = userEditModel.LastName;
             user.SecondName = userEditModel.SecondName;
             user.IsActive = userEditModel.IsActive;
 
             var result = await _usersManager.UpdateAsync(user);
 
-            return Ok(result);
+            var identityErrors = _getErrorIdentityResult(result);
+
+            if (identityErrors != null)
+            {
+                return identityErrors;
+            }
+
+            return Request.CreateResponse(HttpStatusCode.OK);
+        }
+
+        private HttpResponseMessage _getErrorIdentityResult(IdentityResult result)
+        {
+
+            if (result == null)
+            {
+                return Request.CreateErrorResponse(HttpStatusCode.InternalServerError, "Error occured on the server");
+            }
+
+            if (result.Errors.Any())
+            {
+                StringBuilder errorsReport = new StringBuilder();
+
+                foreach (var error in result.Errors)
+                {
+                    errorsReport.Append(error + "; ");
+                }
+
+                return Request.CreateErrorResponse(HttpStatusCode.BadRequest, errorsReport.ToString());
```

Comment:
I am sorry, nevermind. 
When I was reviewing this line looked like it was broken into two lines with wrong format, but then I saw it again.